collect_app/src/main/java/org/odk/collect/android/utilities/ThemeUtils.java

```diff
@@ -108,4 +108,12 @@ public int getAccentColor() {
     public int getIconColor() {
         return getAttributeValue(R.attr.iconColor);
     }
+
+    /**
+     * @return Rank item color for the current {@link android.content.res.Resources.Theme}
+     */
+    @ColorInt
+    public int getRankItemColor() {
+        return getAttributeValue(R.attr.rankItemColor);
```

Comment:
This could be something more generic like "primary background color" or something like that.

Reply:
Fixed.